js_modules/dagit/packages/core/src/app/UserSettingsRoot.tsx

```diff
@@ -18,7 +18,10 @@ import {SHORTCUTS_STORAGE_KEY} from './ShortcutHandler';
 import {TimezoneSelect} from './time/TimezoneSelect';
 import {automaticLabel} from './time/browserTimezone';
 
-const SettingsRoot = () => {
+export interface SettingsRootProps {
+  tabs?: React.ReactNode;
+}
+const UserSettingsRoot: React.FC<SettingsRootProps> = React.memo(({tabs}) => {
```

Comment:
I don't think we can rely on `tabs` being consistently the same object, so this `memo` might not help us much.

If we need some memoization, there might be opportunities elsewhere on this page. Though I don't think of this page as being super expensive.

Reply:
will remove memo for now 👍